src/library/scala/collection/immutable/Vector.scala

```diff
@@ -100,10 +95,7 @@ override def companion: GenericCompanion[Vector] = Vector
     s
   }
 
-
-  // can still be improved
-  override /*SeqLike*/
-  def reverseIterator: Iterator[A] = new AbstractIterator[A] {
+  override /*SeqLike*/ def reverseIterator: Iterator[A] = new AbstractIterator[A] {
```

Comment:
If you don't mind changing the others, may as well change them.  The `override /* SeqLike */` thing is arguably the least important part of the signature, so having it push other stuff out of view on the right, potentially, is a bad idea.  Two lines solve the problem nicely.